ci/release/update-version.sh

```diff
@@ -31,4 +31,8 @@ echo "${NEXT_FULL_TAG}" > VERSION
 
 for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
+  sed_runner "s/:[0-9]*\\.[0-9]*-/:${NEXT_SHORT_TAG}-/g" "${FILE}"
 done
+
+# Update CI image tags of the form {rapids_version}-{something}
+sed_runner "s/:[0-9]*\\.[0-9]*-/:${NEXT_SHORT_TAG}-/g" ./CONTRIBUTING.md
```

Comment:
```suggestion
```

There aren't any references in `CONTRIBUTING.md` like this, so we should remove this.